main_page.php

```diff
@@ -1,19 +1,29 @@
 <?php
+$db = new PDO('mysql:host=db; dbname=collectorapp', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
+//Select all the field in my recipes table and fetch-all to create an assoc array
+$query = $db->prepare('SELECT * FROM `recipes`;');
+$query->execute();
+$recipes = $query->fetchAll();
 ?>
 
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>My Recipes</title>
 </head>
 <body>
-    <section class="recipe_card">
-        <h2>Name:</h2>
-        <h3>Cuisine:</h3>
-        <h3>Time (mins):</h3>
-        <h3>Link to recipe:</h3>
+    <section class= 'recipe_card'>
```

Comment:
Not certain if you were intending this to be outside the foreach loop? This will give you one section for all cards, not one per card as the class would suggest :)

Reply:
That would not be ideal! Thank you :)